Assets/MRTK/Tests/PlayModeTests/ProfileTests.cs

```diff
@@ -98,6 +98,33 @@ private List<IMixedRealityPointer> GetAllPointers()
             }
             return allPointers;
         }
+
+        /// <summary>
+        /// Tests that the camera system initializes with the floor height specified in the profile.
+        /// </summary>
+        /// <returns>enumerator for Unity</returns>
+        [UnityTest]
+        public IEnumerator TestProfileFloorHeight()
```

Comment:
Would it be possible to add a test that involves teleporting?

Reply:
I can write a unit test which covers the teleport + floor height path yeah